src/styles/calendar.scss

```diff
@@ -0,0 +1,23 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 10px;
+  gap: 1px;
+  width: $size-box-day * 7 + 26px;
```

Comment:
1. move 7 to a variable like `week`
2. and what is `26px`? It looks like a `magic number`. Move to a varible as well or calculate like `padding * (week - 1)`